pkg/daemon/daemon.go

```diff
@@ -994,6 +994,9 @@ func (p *ptpProcess) cmdRun(stdoutToSocket bool, pm *PluginManager) {
 					printWhenNotEmpty(logfilter.FilterOutput(p.logFilters, output))
 					p.processPTPMetrics(output)
 					if p.name == ptp4lProcessName {
+						if profileClockType == TBC {
```

Comment:
If this is for when not running cloud-event-proxy then should it have a check for the socket or stdoutToSocket?